src/metadata/use-metadata.js

```diff
@@ -13,9 +13,6 @@ const queryOpts = {
     refetchOnMount: false,
     select: selectorFunction,
     staleTime: 24 * 60 * 60 * 1000,
-    meta: {
```

Comment:
Isn't this needed to persist the metadata between reloads?

Reply:
No, the SW caches it already. The philosophy in this app is:
* Everything that we mutate will be persisted by react-query
* Everything that we fetch only, but never mutate, can be handled by the SW

This way we have a clear separation for when to use what. We only really need the react-query cache when we want to do optimistic updates, which is impossible with the SW. And we don't need to do that for metadata. At least that's my current understanding.

Reply:
Right, it's just that this didn't seem to work as expected, and I got instant load-times when adding this to react-query. It also means that we have more (at least) easier control of the caching - eg. it would be very easy to refetch and clear the cache. I guess it's something we should investigate further, especially regarding e-tags, to have the caching work properly. Lars is aware of that though.

Reply:
Yeah, so ideally SW uses etags to determine whether it should use cached results or not. Right now the SW fetches everything everything while online, it's an offline-support tool. IMO caching the result should be handled through SW/browser caches in the network layer if etags work as expected.

If this is not the case, I could create in issue so we track it as I'd regard that as essential and strongly prefer if we don't add a workaround in the FE and fix it on the server side instead.